primefaces/src/main/java/org/primefaces/renderkit/DataRenderer.java

```diff
@@ -155,7 +155,7 @@ protected void encodePaginatorConfig(FacesContext context, Pageable pageable, Wi
             paginatorContainers = "'" + clientId + "_paginator_top','" + clientId + "_paginator_bottom'";
         }
         else {
-            paginatorContainers = "'" + clientId + "_paginator_" + paginatorPosition + "'";
+            paginatorContainers = "'" + clientId + "_paginator_" + paginatorPosition.toLowerCase() + "'";
```

Comment:
I think we have more or less the same issue with other attributes which basically should have been enums. IMO using an enum would be better (but it would be a huge task to change them all). And if a users uses a value that does not match any of the enum values, we should throw an exception. But again, that's my opinion.

Reply:
I totally agree I noticed we do a ton of `"bottom.equalsIgnoreCase(position)"` type stuff all over but in this one spot it was missed and causes a bug. If it was an enum that would be impossible and we could throw an error.

Maybe it would be worth doing in a bigger cleanup task @tandraschko  ?

Reply:
Should we open a new issue for this?

Reply:
Yes please